packages/yoastseo/src/languageProcessing/helpers/sentence/getSentencesFromTree.js

```diff
@@ -4,11 +4,24 @@
  *
  * @param {Paper} paper The paper to get the sentences from.
  *
- * @returns {Object[]} The array of sentences retrieved from paragraph and heading nodes.
+ * @returns {Sentence[]} The array of sentences retrieved from paragraph and heading nodes plus sourceCodeLocation of the parent node.
  */
 export default function( paper ) {
 	// Get all nodes that have a sentence property which is not an empty array.
 	const tree = paper.getTree().findAll( treeNode => !! treeNode.sentences );
 
-	return tree.flatMap( node => node.sentences );
+	return tree.flatMap( node => node.sentences.map( s => {
```

Comment:
```suggestion
	return tree.flatMap( node => node.sentences.map( sentence => {
```

Any reason you used s instead of sentence?

Reply:
Sharp! I'll change the variable name, it's indeed not clear what `s` is